### DISCUSSION TITLE
GitHub pages now publishes my site as unsecured - http: not https:

### DISCUSSION THREAD

**Discussion by MarkWilcock** [community/community#139715] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Today, GitHub pages is serving up my site as http//. It is no longer https: secured. Yesterday, it was serving up my site as https://

I have made tiny changes to content but nothing that should have forced this. I have also checked the site at https://letsdebug.net/ and there are no issues.  

I have a SSL certificate - it was bought and issued by GoDaddy, where my site was before I moved to GitHub pages

Any ideas / suggestions on what may be the cause and how to fix  gratefully received!

**Comment by 7x9z** [comment]
Hello @MarkWilcock,
You can go to your site settings inside of the repo and enable "Https" in there.
https://github.com/MarkWilcock/reponame/settings/pages
![image](https://github.com/user-attachments/assets/463808c1-1b0f-47c8-8df3-fbfed7b3f55a)

**Comment by azizulabedinazmi** [comment]
Hi @MarkWilcock,

It sounds like you’ve already done quite a bit of troubleshooting! Since your SSL certificate is from GoDaddy and your domain was previously hosted there, it could be a configuration issue between your custom domain and GitHub Pages.

Here are a few steps you can try to resolve the issue:

1. **Check Enforce HTTPS setting**: In your repository settings under **Pages**, make sure the **Enforce HTTPS** option is checked. This ensures that GitHub serves your site over HTTPS.

2. **Update DNS settings**: Verify that your DNS settings are correctly pointing to GitHub Pages. Ensure your domain's A records point to GitHub's IP addresses:
   ```
   185.199.108.153
   185.199.109.153
   185.199.110.153
   185.199.111.153
   ```
   Also, make sure that your CNAME or ALIAS record is set for `www` (or any other subdomain) to `your-username.github.io.`

3. **Verify SSL certificate**: If you're using a custom SSL certificate from GoDaddy, ensure that it is properly installed and valid. Sometimes, switching hosts can require re-issuing or reconfiguring certificates. You may need to check if the certificate is still recognized by GitHub Pages.

4. **Clear browser cache**: Sometimes cached data can cause the site to load over HTTP even after fixing the issue. Clearing your browser’s cache or checking the site in an incognito window might help confirm that the site is properly secured.

If none of these steps work, you might want to check the GitHub Pages support documentation or open a support ticket to further investigate.

Hope this helps!

**Comment by MarkWilcock** [comment]
Thanks for the help.  Much appreciated.  All sorted now.
